### DISCUSSION TITLE
target application not found

### DISCUSSION THREAD

**Discussion by Haocen2004** [LSPosed/LSPosed#1708] [Q&A]
i want to inject `com.miuix.editor`
but i didn't find it on the app list
![Screenshot_2022-02-19-23-30-49-031_com android shell](https://user-images.githubusercontent.com/25221696/154807594-a0faa791-f071-4b79-9f3a-07a99c026f84.jpg)
i found the apk file on `/system/app/MiuixEditor`
![Screenshot_2022-02-19-23-31-02-156_bin mt plus](https://user-images.githubusercontent.com/25221696/154807622-7172f7d6-2134-43ae-968d-2284eff315c9.jpg)

**Comment by yujincheng08** [comment]
We filtered those applications without process. Make sure this application has defined at least one process.

**Reply by Haocen2004** [reply]
this application only provide this action menu (on picture)
can i have some other ways to hook it?
![IMG_20220219_234109](https://user-images.githubusercontent.com/25221696/154807906-78d8ce6d-de2f-4bba-94ab-001722a222c7.jpg)

**Reply by yujincheng08** [reply]
Does it have a process?

**Reply by Haocen2004** [reply]
`  Uid u0a126:
    state=cch
    capability=----
    appWidgetVisible=false
    Package com.miuix.editor:
      RUN_ANY_IN_BACKGROUND (allow): 
        null=[
          Access: [cch-s] 2022-02-19 23:40:26.173 (-39m18s799ms)
        ]`
i can only found it here and in `packageDependencies` args
